orchagent/mirrororch.cpp

```diff
@@ -45,7 +44,7 @@ MirrorEntry::MirrorEntry(const string& platform) :
     nexthopInfo.resolved = false;
     neighborInfo.resolved = false;
 
-    if (platform == MLNX_PLATFORM)
+    if (platform == MLNX_PLATFORM_SUBSTRING)
```

Comment:
Rather than keep changing names to platform string inside PI code, you should come up with a design that removes PD checks from PI code.

Reply:
This change is done according to ACL dynamic configuration design

Reply:
And come from requirements https://github.com/Azure/SONiC/wiki/ACL-Configuration-Requirement-Description

Reply:
I agree with what Nikos said in principle. For this PR, the code is already there. I think we should address the problem in future PR, and we need to more discussion on how to address it properly.